app/models/shipit/deploy.rb

```diff
@@ -66,13 +66,14 @@ def trigger_rollback(user = AnonymousUser.new, env: nil, force: false)
     end
 
     # Rolls the stack back to the **previous** deploy
-    def trigger_revert
+    def trigger_revert(force: false)
```

Comment:
Doesn't need to be fixed here, but `trigger_rollback` would be clearer here, especially now Shipit deals with PRs directly.

Reply:
Yes, unfortunately `trigger_rollback` is already used for something else :/